resources/services/mq/brokers.go

```diff
@@ -274,31 +265,53 @@ func MqBrokers() *schema.Table {
 						Type:        schema.TypeJSON,
 					},
 				},
+				Relations: []*schema.Table{
+					{
+						Name:          "aws_mq_broker_configuration_revisions",
+						Resolver:      fetchMqBrokerConfigurationRevisions,
+						IgnoreInTests: true,
+						Columns: []schema.Column{
+							{
+								Name:        "broker_configuration_cq_id",
+								Description: "Unique CloudQuery ID of aws_mq_broker_configurations table (FK)",
+								Type:        schema.TypeUUID,
+								Resolver:    schema.ParentIdResolver,
+							},
+							{
+								Name:        "configuration_id",
+								Description: "Required",
+								Type:        schema.TypeString,
+							},
+							{
+								Name:        "created",
+								Description: "Required",
+								Type:        schema.TypeTimestamp,
+							},
+							{
+								Name:        "data",
+								Description: "Required",
+								Type:        schema.TypeJSON,
+								Resolver:    resolveBrokerConfigurationRevisionsData,
+							},
+							{
+								Name:        "description",
+								Description: "The description of the configuration.",
+								Type:        schema.TypeString,
+							},
+						},
+					},
+				},
 			},
 			{
-				Name:        "aws_mq_broker_users",
-				Resolver:    fetchMqBrokerUsers,
-				IgnoreError: client.IgnoreAccessDeniedServiceDisabled,
-				Options:     schema.TableCreationOptions{PrimaryKeys: []string{"broker_cq_id", "username"}},
+				Name:     "aws_mq_broker_users",
+				Resolver: fetchMqBrokerUsers,
```

Comment:
PK removed?

Reply:
as we don't use manual PKs on relations I have removed them. They are created automatically if not set